source/_integrations/sql.markdown

```diff
@@ -131,6 +131,67 @@ For more detailed steps on how to define a custom interval, follow the procedure
 
 {% include common-tasks/define_custom_polling.md %}
 
+## Actions
+
+### Action `sql.query`
+
+This action allows you to execute an arbitrary read-only `SELECT` query against a database and get the results back.
+
+| Action data attribute | Optional | Description                                                                                                        |
```

Comment:
We are trying to use lists instead of tables, where possible.
https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L139

https://developers.home-assistant.io/docs/documenting/general-style-guide#tables